lib/hawkular/base_client.rb

```diff
@@ -139,10 +139,11 @@ def normalize_entrypoint_url(entrypoint, suffix_path)
       strip_path.nil? || suffix_path = strip_path
       strip_path = suffix_path.gsub(%r{^/}, '')
       strip_path.nil? || suffix_path = strip_path
-      strip_entrypoint = entrypoint.gsub(%r{/$}, '')
+      the_entry_point = (entrypoint.is_a? URI::HTTP) ? entrypoint.to_s : entrypoint
```

Comment:
On 8 Jul 2016, at 14:02, Oleg Barenboim wrote:
> Why not the simpler `entrypoint = entrypoint.to_s`?

Isn't a to_s on something that is already a string a bit odd? But sure I 
can change that.